roles/ssp/templates/config/authsources-1.14.php.j2

```diff
@@ -22,7 +26,10 @@ $config = array(
         // Can be NULL/unset, in which case an entity ID is generated based on the metadata URL.
         'entityID' => '{{ authsource.entity_id }}',
 {% endif %}
-{% if authsource.idp is defined %}
+{% if authsource.idp_function is defined %}
+        // The entity ID of the IdP constructed dynamically using a function.
```

Comment:
Replace with:
// The function returning the entity ID of the IdP this SP should contact.
// Can be NULL/unset, in which case the user will be shown a list of available IdPs.

Reply:
Done